suite/tests/client-interface/stolen-reg.dll.c

```diff
@@ -67,6 +67,59 @@ do_flush(app_pc next_pc)
     DR_ASSERT(false);
 }
 
+// Storing the original stolen reg before mconext set.
+static ptr_int_t orig_value = 0;
+
+// Random value to detect after mcontext set.
+static ptr_int_t test_value = 7;
+
+static void
+load_stolen_reg_to_mcontext()
+{
+    fprintf(stderr, "load_stolen_reg_to_mcontext entered\n");
```

Comment:
suggest: Avoid duplication to avoid stale values when one changes: use `%s` and `__FUNCTION__`.

Reply:
Done.